frontend/src/scenes/authentication/signup/verify-email/VerifyEmail.tsx

```diff
@@ -17,28 +19,75 @@ export const VerifyEmailHelpLinks = (): JSX.Element => {
     const { requestVerificationLink } = useActions(verifyEmailLogic)
     const { uuid } = useValues(verifyEmailLogic)
     const { openSupportForm } = useActions(supportLogic)
+    const [iNeedHelp, setINeedHelp] = useState(false)
+    const [checkListValues, setCheckListValues] = useState<boolean[]>([])
+
+    if (!iNeedHelp) {
+        return (
+            <LemonButton type="secondary" className="mt-2" onClick={() => setINeedHelp(true)}>
+                I need help
+            </LemonButton>
+        )
+    }
+
+    const checklist = ['Checked your spam folder', 'Checked with your IT department (if applicable)']
```

Comment:
- [ ] Wait 5 minutes. Sometimes it takes a bit for ISPs to deliver emails.
- [ ] Check your spam folder
- [ ] Check any firewalls you may have active
- [ ] Ask your company IT department to allow any emails from `@posthog.com`
- [ ] Make sure you can receive other emails to your email address, especially if it's an alias
- [ ] Make an offering to the hedgehog gods and check your email again

The last one is a joke but not really, it might make some people laugh, might offend some others of the monotheism type lol